### PR TITLE
fix: move to AWS Code Artifact

### PR DESCRIPTION
**Pull request by erlend-vimond**
## *What* does this PR do?
- Moved to AWS code artifact
- Bumped minor version numbers
- Upgraded to Node 20
- Upgraded dependencies
- Pinned node version

## JIRA Tickets
https://vimond-ng.atlassian.net/browse/PLAT-1513

## Manual Testing
Tested [here](https://github.com/vimond/drifter/pull/30):
- Push: https://github.com/vimond/drifter/actions/runs/10662655772/job/29550260750
- PR: https://github.com/vimond/drifter/actions/runs/10662680253/job/29550325269?pr=30

### CODE REVIEW

File: .mise.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[tools]
+node = "16"
```
Comment (dagvl) [inline .mise.toml:2]: should we upgade the version? EOL was 11 Sep 2023

File: .github/workflows/ticket-storer-push.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ jobs:
 
       - name: store valid tickets
         id: store
-        uses: vimond/actions/ticket-storer@0.0.1
+        uses: vimond/actions/ticket-storer@fix/jfrog
```
Comment (dagvl) [inline .github/workflows/ticket-storer-push.yaml:29]: remember to fix these again when merging

**Review by dagvl — APPROVED** [review]
Looks good! Minor comments.

QA: The test of these are in the usage as they will be used in all our github repos. 

image

File: .mise.toml (hunk 1 above)
Comment (erlend-vimond) [reply .mise.toml]: Done 👍  I had to upgrade some dependencies too. Also, we didn't pin the node version in the actions, so we were relying on the system default, which is periodically updated. I moved to using the `setup-node` step to pin it, which is recommended.

File: .github/workflows/ticket-storer-push.yaml (hunk 2 above)
Comment (erlend-vimond) [reply .github/workflows/ticket-storer-push.yaml]: Done, reverted 👍

**Pull request merged**